dev_tools/requirements/isolated-base.env.txt

```diff
@@ -4,4 +4,4 @@
 -r deps/notebook.txt
 
 # for shell_tools
-requests
+requests>=2.32
```

Comment:
For consistency, let us use the same requirement everywhere.  It is also better to pin the major version so we guard against incompatible changes in their APIs.

```suggestion
requests~=2.32
```